Example facet grid

Shee-Ra opened the pull request:
I created an example demonstrating how to use facet_grid.

First time contributing to a project, if I need to fix anything please let me know.

Shee-Ra committed:
built facet_grid first draft example

Shee-Ra committed:
proofed writing

Shee-Ra committed:
Update contributors.txt

has2k1 commented:
Nice. Can you make two changes,

1. Replace `+ xlab()` and `+ ylab()` with `+ labs(x='...', y='...')`. `xlab` & `ylab` are compatibility functions carried over from base R to ggplot2 to plotnine.
2. Print dataframe after adding the 'transmission' column, it allows readers to just see the result of the data manipulation instead of trying to understand the syntax.
```python
mpg["transmission"] = mpg['trans'].map(lambda x: "auto" if "auto" in x else "man" if "man" in x else "")
```

Shee-Ra committed:
used labs for labeling, and added other minor updates to facet_grid example

Shee-Ra commented:
Thanks for the feedback. I updated as requested, and fixed up some text I noticed could be clearer. I updated my branch so I guess it's good to go? Cheers.
